en/projects/cie.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Electronic Identity Card (CIE)
+lang: en
+permalink: /en/projects/cie.htm
+layout: project
+ref: cie
+parent_ref: projects
+people: [valerio-paolini]
+toc: true
+medium_tag: cie
+forum_category: cie
+forum_limit: 3
+twitter_tag: cie
+tweetdeck_id: 913425127978696706
+dashboard_url: https://dashboard.teamdigitale.governo.it/public/dashboard/e155f9b3-7624-4e62-8b29-25e9f7d6dad5
+---
+
+### In short
+
+CIE, the Italian Electronic Identity Card, is a highly secure electronic document, equipped with a radio frequency microprocessor that enables it for various use cases, including access to online services. CIE complies with security regulations and substitutes the previous paper format.
+
+### The benefits for citizens and the public administration
+
+CIE is a document not subject to counterfeiting and allows secure access to physical and digital services.
```

Comment:
CIE is a document not subject to counterfeiting -> CIE is safe/secured against counterfeiting attempts (?)

Reply:
I'd go for 'CIE is secured against counterfeiting attempts'